_cffi_build/build.py

```diff
@@ -36,14 +34,13 @@ def _mk_ffi(sources, name="_libevmjit", bundled=True, **kwargs):
 _base = [Source(absolute("evm.h"), "#include \"_cffi_build/evm.h\"",)]
 ffi = _mk_ffi(_base, libraries=['evmjit'])
 ffi.cdef("""
-    extern "Python" void  evm_query(struct evm_env* env,
+    extern "Python" union evm_variant evm_query(struct evm_env* env,
                                           enum evm_query_key key,
-                                          union evm_variant* arg,
-                                          union evm_variant* ret);
+                                          union evm_variant arg);
```

Comment:
Seems like the cffi does not guarantee that it will translate such code correctly. Did you find any reference in the cffi docs?